### ISSUE TITLE
backup_requirement.txt

### ISSUE THREAD

**Issue by seirah-yang** [seirah-yang/libraries.zip#1]
absl-py==2.3.1
accelerate==1.10.1
aiohappyeyeballs==2.6.1
aiohttp==3.12.15
aiosignal==1.4.0
astunparse==1.6.3
async-timeout==5.0.1
attrs @ file:///croot/attrs_1734533101012/work
backcall @ file:///home/conda/feedstock_root/build_artifacts/backcall_1592338393461/work
Bottleneck @ file:///croot/bottleneck_1731058641041/work
Brotli @ file:///croot/brotli-split_1736182456865/work
cachetools==5.5.2
certifi @ file:///croot/certifi_1754570635119/work/certifi
charset-normalizer @ file:///croot/charset-normalizer_1721748349566/work
cloudpickle==3.1.1
comm @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_comm_1753453984/work
contourpy @ file:///croot/contourpy_1738160616259/work
cycler @ file:///tmp/build/80754af9/cycler_1637851556182/work
datasets==4.1.1
debugpy @ file:///home/task_175706710612866/conda-bld/debugpy_1757067121768/work
decorator @ file:///home/conda/feedstock_root/build_artifacts/decorator_1740384970518/work
dill==0.4.0
dm-tree==0.1.8
faiss-gpu-cu11==1.12.0
fastjsonschema @ file:///croot/python-fastjsonschema_1731939362158/work
filelock==3.19.1
flatbuffers==25.9.23
fonttools @ file:///home/task_175879159822283/conda-bld/fonttools_1758791613454/work
frozenlist==1.7.0
fsspec==2025.9.0
gast==0.4.0
google-auth==2.40.3
google-auth-oauthlib==1.0.0
google-pasta==0.2.0
grpcio==1.74.0
h5py==3.14.0
hf-xet==1.1.10
huggingface-hub==0.35.0
idna==3.10
importlib_metadata @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_importlib-metadata_1747934053/work
importlib_resources @ file:///croot/importlib_resources-suite_1720641103994/work
ipykernel @ file:///home/conda/feedstock_root/build_artifacts/ipykernel_1754352855579/work
ipython @ file:///home/conda/feedstock_root/build_artifacts/ipython_1651240551705/work
jedi @ file:///home/conda/feedstock_root/build_artifacts/jedi_1733300866624/work
Jinja2==3.1.6
joblib @ file:///home/task_175792628848965/conda-bld/joblib_1757926345058/work
jsonschema @ file:///croot/jsonschema_1753206402902/work
jsonschema-specifications @ file:///croot/jsonschema-specifications_1699032386549/work
jupyter_client @ file:///home/conda/feedstock_root/build_artifacts/jupyter_client_1733440914442/work
jupyter_core @ file:///home/conda/feedstock_root/build_artifacts/jupyter_core_1748333051527/work
keras==2.13.1
kiwisolver @ file:///croot/kiwisolver_1672387140495/work
libclang==18.1.1
lxml @ file:///croot/lxml_1737039601731/work
Markdown==3.9
markdown-it-py==3.0.0
MarkupSafe==3.0.2
matplotlib==3.9.2
matplotlib-inline @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-inline_1733416936468/work
mdurl==0.1.2
mkl-service==2.4.0
ml-dtypes==0.2.0
mpmath==1.3.0
multidict==6.6.4
multiprocess==0.70.16
namex==0.1.0
narwhals @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_narwhals_1753426042/work
nbformat @ file:///croot/nbformat_1728049424075/work
nest_asyncio @ file:///home/conda/feedstock_root/build_artifacts/nest-asyncio_1733325553580/work
networkx==3.2.1
numpy @ file:///home/conda/feedstock_root/build_artifacts/numpy_1707225342954/work/dist/numpy-1.26.4-cp39-cp39-linux_x86_64.whl#sha256=c799942b5898f6e6c60264d1663a6469a475290e758c654aeeb78e2596463abd
nvidia-cublas-cu11==11.11.3.6
nvidia-cublas-cu12==12.1.3.1
nvidia-cuda-cupti-cu12==12.1.105
nvidia-cuda-nvrtc-cu12==12.1.105
nvidia-cuda-runtime-cu11==11.8.89
nvidia-cuda-runtime-cu12==12.1.105
nvidia-cudnn-cu12==8.9.2.26
nvidia-cufft-cu12==11.0.2.54
nvidia-curand-cu12==10.3.2.106
nvidia-cusolver-cu12==11.4.5.107
nvidia-cusparse-cu12==12.1.0.106
nvidia-nccl-cu12==2.19.3
nvidia-nvjitlink-cu12==12.9.86
nvidia-nvtx-cu12==12.1.105
oauthlib==3.3.1
opt_einsum==3.4.0
optree==0.17.0
packaging @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_packaging_1745345660/work
pandas @ file:///home/conda/feedstock_root/build_artifacts/pandas_1702057139790/work
parso @ file:///croot/parso_1733963305961/work
pexpect @ file:///home/conda/feedstock_root/build_artifacts/pexpect_1733301927746/work
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1733327343728/work
pillow @ file:///croot/pillow_1744613067434/work
platformdirs @ file:///home/conda/feedstock_root/build_artifacts/platformdirs_1715777629804/work
plotly @ file:///home/conda/feedstock_root/build_artifacts/plotly_1755067229108/work
polars==1.2.1
pooch @ file:///croot/pooch_1731007038704/work
prompt_toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1744724089886/work
propcache==0.3.2
protobuf==3.20.3
psutil @ file:///home/task_175710407404039/conda-bld/psutil_1757104540576/work
ptyprocess @ file:///home/conda/feedstock_root/build_artifacts/ptyprocess_1733302279685/work/dist/ptyprocess-0.7.0-py2.py3-none-any.whl#sha256=92c32ff62b5fd8cf325bec5ab90d7be3d2a8ca8c8a3813ff487a8d2002630d1f
pyarrow==21.0.0
pyasn1==0.6.1
pyasn1_modules==0.4.2
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1750615794071/work
pyparsing @ file:///croot/pyparsing_1731445506121/work
PySocks @ file:///tmp/build/80754af9/pysocks_1605305812635/work
python-dateutil @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_python-dateutil_1751104122/work
python-docx @ file:///home/conda/feedstock_root/build_artifacts/python-docx_1750157474613/work
pytz @ file:///croot/pytz_1752135852232/work
PyYAML==6.0.2
pyzmq @ file:///home/task_175810350423983/conda-bld/pyzmq_1758103934837/work
rank-bm25==0.2.2
referencing @ file:///croot/referencing_1699012038513/work
regex==2025.9.18
requests @ file:///croot/requests_1756709366904/work
requests-oauthlib==2.0.0
rich==14.1.0
rpds-py @ file:///croot/rpds-py_1736541261634/work
rsa==4.9.1
safetensors==0.6.2
scikit-learn @ file:///croot/scikit-learn_1737988764076/work
scipy @ file:///croot/scipy_1733756309941/work/dist/scipy-1.13.1-cp39-cp39-linux_x86_64.whl#sha256=3b247b926209f2d9f719ebae39faf3ff891b2596150ed8f8349adfc3eb19441c
seaborn @ file:///croot/seaborn_1749110291192/work
sentence-transformers==5.1.1
six @ file:///home/conda/feedstock_root/build_artifacts/bld/rattler-build_six_1753199211/work
sympy==1.14.0
tensorboard==2.13.0
tensorboard-data-server==0.7.2
tensorflow==2.13.1
tensorflow-estimator==2.13.0
tensorflow-io-gcs-filesystem==0.37.1
tensorflow-probability==0.21.0
termcolor==3.1.0
threadpoolctl @ file:///croot/threadpoolctl_1719407800858/work
tokenizers==0.22.1
torch==2.2.2
tornado @ file:///croot/tornado_1748956929273/work
tqdm==4.67.1
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1733367359838/work
transformers==4.56.2
triton==2.2.0
typeguard==2.13.3
typing_extensions @ file:///croot/typing_extensions_1756280817316/work
tzdata @ file:///croot/python-tzdata_1746123641790/work
unicodedata2 @ file:///croot/unicodedata2_1736541023050/work
urllib3 @ file:///croot/urllib3_1737133630106/work
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1733231326287/work
Werkzeug==3.1.3
wrapt==1.17.3
xxhash==3.5.0
yarl==1.20.1
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1749421620841/work
